src/cljs/athens/views.cljs

```diff
@@ -61,10 +64,9 @@
        [:div [:a {:href (rfee/href :pages)} "All /pages"]]
        [:div [:span {:style {}} "Current Route: " [:b (-> @current-route :path)]]]
        [:div {:style {:border-bottom "1px solid gray" :margin "10px 0"}}]
-       [:ol {:style {:padding 0 :margin 0 :list-style-type "none"}}
+       [:ul {:class "left-sidebar"}
```

Comment:
Instead of adding classes to extract the css. Maybe we can add functions so we don't have to think about classes? E.g. we could have a function in styles +sidebar. We would reference it here as `style/+sidebar`.

If we use styles or classes it wouldn't matter. We can swap the implementation there, e.g.
```clojure
(defn +sidebar [attrs]
  (update attrs :style merge {:padding 0 :margin 0 :list-style-type "none"}))

(defn +sidebar [attrs]
  (update attrs :class (partial str " ") "left-sidebar"))
```

With `carve` in #55 we can remove unused style functions and keep styles.cljs small. What do you think?